### ISSUE TITLE
Binary release of VCPKG?

### ISSUE THREAD

**Issue by breznak** [mattheiler/nupic.core-redux#1]
I really like this dependency management reduction! 
VCPKG looks very promising, but is there a chance for more "user friendly" version - a binary package? 

I think this would be not convenient for many non-programmer users: 

> Clone this repository, then run
C:\src\vcpkg> .\bootstrap-vcpkg.bat
Then, to hook up user-wide integration, run (note: requires admin on first use)
C:\src\vcpkg> .\vcpkg integrate install
Install any packages with
C:\src\vcpkg> .\vcpkg install sdl2 curl
Finally, create a New Project (or open an existing one) in Visual Studio 2017 or 2015. All installed libraries are immediately ready to be #include'd and used in your project.

**Comment by mattheiler** [comment]
I imagine a script could be written to install vcpkg, if Git for Windows were installed as a perquisite. This would be for the developer environment, though - is there a use case for non-programmers in a development environment? We would repackage and redistribute any dependencies. What's missing is copying any of the found binaries into a local bin folder.

**Comment by breznak** [comment]
> This would be for the developer environment, though - is there a use case for non-programmers in a development environment? 

Good point, I didn't understand that. For devs it's imho OK to install git, even vcpkg. Unless non-prog users can install binary release.

**Comment by ras0219-msft** [comment]
> What's missing is copying any of the found binaries into a local bin folder.

Vcpkg does have the capability of exporting out freestanding binary packages (including nupic and all dependencies) that will handle this for both MSBuild and CMake (and others, with work). I'd be happy to answer any questions about it!

**Comment by mattheiler** [comment]
Yes, I was hoping that feature would cover it! Might look into that tonight. I'll reach out if I have any questions.
